src/app/photographer/[handle]/layout.tsx

```
@@ -0,0 +1,57 @@
+import type { Metadata } from "next";
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)